components/footerComponent.tsx

```diff
@@ -68,14 +36,6 @@ export default function FooterComponent() {
             </div>
           </Popover>
         </nav>
-        <div className="flex space-x-6">
-          {navigation.social.map((item) => (
-            <a key={item.name} target="_blank" href={item.href} className={`text-primary-text hover:text-gray-400 ${item.className}`}>
```

Comment:
Is this footer still used?

Reply:
nop, I created new globalFooter file for this:
![image](https://github.com/layerswap/layerswapapp/assets/60259252/21be9dee-b30d-4f9a-af86-fc400f28dab4)